### DISCUSSION TITLE
Single Sign-On (SSO) - user saved to the database, but server fails on cookie problem

### DISCUSSION THREAD

**Discussion by Alex-NHS** [directus/directus#17910] [Archive]
Hi everyone,

I am developing an internal app using Directus v9.23.3

I configured SSO with Azure and Google, and both were working fine, users were sent to Google and Microsoft for authentication, then the user was redirected back to Directus, but then the server crashed, below is the message:

![image](https://user-images.githubusercontent.com/109280921/227423871-3eda524c-a9bc-49b1-a553-6612e01fa19b.png)

I used a default login to check the user collection & notice that the user details are actually got stored on the database (image below)

![image](https://user-images.githubusercontent.com/109280921/227424194-4a4ca57e-6cc6-49ef-a793-047a3ea1402d.png)

I have got REFRESH_TOKEN_COOKIE_SECURE already set to false as well

![image](https://user-images.githubusercontent.com/109280921/227425757-2f94d988-a714-45dc-b98f-394bc3e1df9b.png)

Help and guidance are greatly appreciated! Thank you

**Comment by Zack-Heisnberg** [comment]
the issue was resolved in the latest version

**Reply by Alex-NHS** [reply]
thank you!!!, it works now, i will close this discussion
